lib/smack/SmackRep.cpp

```diff
@@ -76,8 +137,20 @@ string SmackRep::type(const llvm::Type* t) {
     return BOOL_TYPE;
   else if (isFloat(t))
     return FLOAT_TYPE;
-  else
+  else if (isInt(t)) {
+    if (BIT_VECTOR) {
+      stringstream s;
+      s << "i" << getIntSize(t);
+      return s.str();
```

Comment:
I'll work on it while I have a slightly irelevant question that SmackRep has a lot of functions returning a string which may lead to a number of string copies. But String copy is kind of expensive.